network-api/networkapi/management/commands/fix_redirects.py

```diff
@@ -0,0 +1,56 @@
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
+from wagtail.contrib.redirects.models import Redirect
+from wagtail.models import Locale
+
+# Define the list of valid values to check against
+LOCALES = Locale.objects.all().values_list("language_code", flat=True)
+
+
+def check_path(url_path):
```

Comment:
What do you think about naming the function after the check it performs? Something like `is_starting_with_language_code`?

Reply:
:+1: adjusted in d2b0cabafe3f555bb55786c6d77bedb793ea88fb